configs/DRONEERH743DUAL/config.h

```diff
@@ -0,0 +1,183 @@
+#define GYRO_2_SPI_INSTANCE             SPI4
+#define GYRO_2_ALIGN                    CW180_DEG
+
+// Blackbox
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define SPI3_SCK_PIN                    PC10 //SPI3_SCK/USART3_TX/UART4_TX/SDMMC1_D2
+#define SPI3_SDI_PIN                    PC11 //SPI3_MISO/USART3_RX/UART4_RX/SDMMC1_D3
+#define SPI3_SDO_PIN                    PC12 //SPI3_MOSI/UART5_TX/SDMMC1_CK
+#define FLASH_CS_PIN                    PA8  //TIM1_CH1/I2C3_SCL/UART7_RX
+#define FLASH_SPI_INSTANCE              SPI3
+#define DEFAULT_BLACKBOX_DEVICE         BLACKBOX_DEVICE_FLASH
+
+// BAT voltage ADC
+#define ADC_VBAT_PIN                    PC0  //ADC123_INP10
+#define DEFAULT_VOLTAGE_METER_SOURCE    VOLTAGE_METER_ADC
+// #define DEFAULT_VOLTAGE_METER_SCALE     210
+
+// BAT current ADC
+#define ADC_CURR_PIN                    PC1  //SPI2_MOSI/ADC123_INN10/ADC123_INP11
+#define DEFAULT_CURRENT_METER_SOURCE    CURRENT_METER_ADC
+// #define DEFAULT_CURRENT_METER_SCALE     120
+
+// MAX7456 OSD
+#ifdef USE_OSD_SD
+#define USE_MAX7456
+#define SPI2_SCK_PIN                    PB13 //TIM1_CH1N/SPI2_SCK/USART3_NSS/UART5_TX
+#define SPI2_SDI_PIN                    PB14 //TIM1_CH2N/TIM12_CH1/TIM8_CH2N/USART1_TX/SPI2_MISO
+#define SPI2_SDO_PIN                    PB15 //TIM1_CH3N/TIM12_CH2/TIM8_CH3N/USART1_RX/SPI2_MOSI
+#define MAX7456_SPI_CS_PIN              PB12 //I2C2_SMBA/SPI2_NSS/UART5_RX
+#define MAX7456_SPI_INSTANCE            SPI2
+#endif
+
+// Debug
+#define USE_SWDIO_PIN                   PA13
+#define USE_SWCLK_PIN                   PA14
+
+// Motor configuration
+#define MOTOR1_PIN                      PE9  //TIM1_CH1
+#define MOTOR2_PIN                      PE11 //TIM1_CH2/SPI4_NSS
+#define MOTOR3_PIN                      PE13 //TIM1_CH3/SPI4_MISO
+#define MOTOR4_PIN                      PE14 //TIM1_CH4/SPI4_MOSI
+#define MOTOR5_PIN                      PA6  //TIM3_CH1/SPI1_MISO/SPI6_MISO/TIM13_CH1/ADC12_INP3
+#define MOTOR6_PIN                      PA7  //TIM3_CH2/SPI1_MOSI/SPI6_MOSI/TIM14_CH1/ADC12_INN3
+#define MOTOR7_PIN                      PC8  //TIM3_CH3/TIM8_CH3/SDMMC1_D0
+#define MOTOR8_PIN                      PC9  //TIM3_CH4/TIM8_CH4/I2C3_SDA/SDMMC1_D1
+
+// TIMER_PIN_MAP(idx, pin, occurance, dma_opt)
+#define TIMER_PIN_MAPPING               TIMER_PIN_MAP( 0, MOTOR1_PIN, 1,  0 ) \
+                                        TIMER_PIN_MAP( 1, MOTOR2_PIN, 1,  1 ) \
+                                        TIMER_PIN_MAP( 2, MOTOR3_PIN, 1,  2 ) \
+                                        TIMER_PIN_MAP( 3, MOTOR4_PIN, 1,  3 ) \
+                                        TIMER_PIN_MAP( 4, MOTOR5_PIN, 1,  4 ) \
+                                        TIMER_PIN_MAP( 5, MOTOR6_PIN, 1,  5 ) \
+                                        TIMER_PIN_MAP( 6, MOTOR7_PIN, 1,  6 ) \
+                                        TIMER_PIN_MAP( 7, MOTOR8_PIN, 1,  7 ) \
+                                        TIMER_PIN_MAP( 8, LED_STRIP_PIN, 1, 11 )
```

Comment:
```suggestion
                                        TIMER_PIN_MAP( 8, LED_STRIP_PIN, 2, 11 ) \
                                        TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 3, -1 ) \
                                        TIMER_PIN_MAP(10, GYRO_1_CLKIN_PIN, 1, -1 ) \
                                        TIMER_PIN_MAP(11, GYRO_2_CLKIN_PIN, 2, -1 ) \
                                        TIMER_PIN_MAP(12, BEEPER_PIN, 1, -1 )
```

Reply:
Note GYRO_2_CLKIN_PIN conflicts with motors 5&6, but this should not be an issue when using 4 motors or when bitbanged DShot is used. We can also disable CLKIN for gyro 2 instead.